src/tests/pages/place/[placeId]/status.test.js

```diff
@@ -0,0 +1,76 @@
+import { place as mockedPlace, place } from '../../../data/place';
+import { parseOfferId } from '../../../../utils/parseOfferId';
+import { setupPage } from '../../../utils/setupPage';
+
+import { waitFor, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import Status from '../../../../pages/place/[placeId]/status';
+import nl from '../../../../i18n/nl.json';
+
+import { renderPageWithWrapper } from '../../../utils/renderPageWithWrapper';
+import { waitForFetch } from '../../../utils/waitForFetch';
+
+describe('Status page place', () => {
+  describe('When the status "Available" is loaded from place object', () => {
+    let router;
+
+    beforeEach(async () => {
+      router = setupPage({
```

Comment:
maybe we should separate setupPage into setupRouter which returns the router object and setupPage or a different name that handles the rest

Reply:
It would be more logical when the router object is returned from a function called `setupRouter` or `createMockRouter`

Reply:
Yeah but setupPage does more than that. 
If you split it up, you need to remember to do all the steps every time.

Reply:
I could return an object that contains the router?

Reply:
and doesn't `createMockRouter` fit the function `setupPage`

Reply:
it also sets the mockresponses

Reply:
Yep like this approach 👍🏻